lib/widgets/davnavbar/DayButton.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+
+const colors = [
+  Color.fromARGB(255, 0, 0, 0),
+  Color.fromARGB(255, 150, 150, 150),
+  Color.fromARGB(255, 180, 180, 180)
+];
+
+class DayButton extends StatelessWidget {
+  final Function setCurrentDate;
+  final DateTime date;
+  final int position;
+
+  const DayButton(
+      {Key? key,
+      required this.date,
+      required this.setCurrentDate,
+      required this.position})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialButton(
+      color: const Color.fromARGB(255, 240, 233, 225),
+      shape: const CircleBorder(),
+      onPressed: () {
+        debugPrint(date.toString());
+        setCurrentDate(date);
+      },
+      child: Text(
+        '${date.day}',
```

Comment:
Do you think showing date.weekday, or more realistically, an initial of the day of the week based on date.weekday, would be helpful here? Definitely not a "need to have," but I think down the road it'd be a "nice to have"

Reply:
I'm going to put this on the other MR since it is easier to do with the date formatting utils I brought it for making the calendar widget.